test/odbc/constants.cpp

```diff
@@ -1,6 +1,11 @@
 #include "constants.h"
 using std::string;
 
+map<constants::ServerType, string> constants::server_to_odbc_types = { 
+    {ServerType::MSSQL, "MSSQL"}, 
+    {ServerType::PSQL, "PSQL"} 
+};
+
 string constants::INPUT_SOURCE = "input";
```

Comment:
We should probably delete the rest of these constants if they aren't in use anymore.

Reply:
Done.